src/main/java/net/pms/dlna/MediaMonitor.java

```diff
@@ -206,49 +206,44 @@ public void stopped(DLNAResource res) {
 						LOGGER.debug("An error occurred when dumping monitor file: " + e);
 					}
 
-					File watchedFile = new File(rf.getFile().getAbsolutePath());
+					File playedFile = new File(rf.getFile().getAbsolutePath());
 
 					if (fullyPlayedAction == 3) {
 						// Move the video to a different folder
-						String newDirectory = configuration.getFullyPlayedOutputDirectory();
-						if (!newDirectory.endsWith("\\")) {
-							newDirectory += "\\\\";
-						}
+						String newDirectory = FileUtil.appendPathSeparator(configuration.getFullyPlayedOutputDirectory());
 
-						try {
-							if (watchedFile.renameTo(new File(newDirectory + watchedFile.getName()))) {
-								LOGGER.debug("Moved {} because it has been fully played", watchedFile.getName());
-							} else {
-								LOGGER.debug("Moving {} failed, trying again in 3 seconds", watchedFile.getName());
+						if (playedFile.renameTo(new File(newDirectory + playedFile.getName()))) {
+							LOGGER.debug("Moved {} because it has been fully played", playedFile.getName());
+						} else {
+							LOGGER.debug("Moving {} failed, trying again in 3 seconds", playedFile.getName());
 
-								try {
-									Thread.sleep(3000);
-								} catch (InterruptedException e) {
-									LOGGER.error(null, e);
-								}
+							try {
+								Thread.sleep(3000);
 
-								if (watchedFile.renameTo(new File(newDirectory + watchedFile.getName()))) {
-									LOGGER.debug("Moved {} because it has been fully played", watchedFile.getName());
+								if (playedFile.renameTo(new File(newDirectory + playedFile.getName()))) {
+									LOGGER.debug("Moved {} because it has been fully played", playedFile.getName());
 								} else {
-									LOGGER.info("Failed to move {}", watchedFile.getName());
+									LOGGER.info("Failed to move {}", playedFile.getName());
 								}
+							} catch (InterruptedException e) {
```

Comment:
I changed the InterruptedException behaviour as that is not an error but a normal occurrence when UMS is shut down (all threads get interrupt signals then, which means they should hurry up and finish what they're doing). Any caught interrupt should be reenabled so that parent objects also knows it's time to quit.